### DISCUSSION TITLE
【支持脚注】同页面内block link跳转到指定位置

### DISCUSSION THREAD

**Discussion by zfhxi** [tangly1024/NotionNext#2056] [Ideas]
**描述bug**

在我的文章中，有些文字的描述是参考了一些文献的，所以我希望在引用处添加脚注。
然而Notion不支持脚注，我是通过"Copy link to block"获取块访问链接，然后在引用处添加这个链接，比如[这篇文章](https://foil-saturnalia-95a.notion.site/Sinkhorn-Distance-9560982f02c54d45a77a735537dcd39e#31c8a137923a453aa2970a4b0960be38)的“文献[1]”添加了链接[https://foil-saturnalia-95a.notion.site/Sinkhorn-Distance-9560982f02c54d45a77a735537dcd39e#f50f688882954dd1b170467685cef676](https://foil-saturnalia-95a.notion.site/Sinkhorn-Distance-9560982f02c54d45a77a735537dcd39e#f50f688882954dd1b170467685cef676)。

然而部署后，上述link被NotionNext渲染为：
```html
<a class="notion-link" href="/9560982f02c54d45a77a735537dcd39e#f50f688882954dd1b170467685cef676">[1]</a>
```
这样点击“文献[1]”，实际是打开了新页面：`https://idzc.top/9560982f02c54d45a77a735537dcd39e#f50f688882954dd1b170467685cef676`。

**复现步骤**
见**描述bug**

**期望的正常结果**
期望NotionNext渲染成:
```html
<a class="notion-link" href="#f50f688882954dd1b170467685cef676">[1]</a>
```
并且对应的block（第一篇参考文献位置所在的block）可以渲染为id="#f50f688882954dd1b170467685cef676"的元素。

这样点击“文献[1]”，将跳转到：
`https://idzc.top/article/202305042300#f50f688882954dd1b170467685cef676`。

**截图**
无

**环境**

- 操作系统: [macOS]
- 浏览器 [例如. chrome]
- 版本 [Version 112.0.5615.121 (Official Build) (arm64)]

**补充说明**
暂无

**Comment by tangly1024** [comment]
> 1. 对应的block（第一篇参考文献位置所在的block）可以渲染为id="#f50f688882954dd1b170467685cef676"的元素。
将文献格式设置为 header（建议设置为二级标题），即可实现 id="#f50f688882954dd1b170467685cef676"

>  "Copy link to block"获取块访问链接，然后在引用处添加这个链接
获取后的链接需要加工一下 保留 #号开始的内容
![image](https://github.com/tangly1024/NotionNext/assets/15920488/5c9bd3bd-3a93-4237-9c50-4adbf28e23eb)

不过，点击文献[1]，会在新窗口中打开，而不是在当前页面跳转，要实现该功能，需要改动代码
